components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react"
+import type { ReactNode } from "react"
+import classNames from "classnames"
+
+import styles from "./Tabs.module.scss"
+/**
+ *
+ * @todo implement left-right arrow navigation of tabs:
+ * https://dev.to/eevajonnapanula/keyboard-accessible-tabs-with-react-5ch4
+ */
+
+interface ITabsProps {
+  tabLabels: string[]
+  // is this the right way to type react children?
+  children: ReactNode[]
+}
+
+interface ITabButtonProps {
+  isOpen: boolean
+  label: string
+  id: string
+  onTabClick: (label: string) => void
+}
+
+interface ITabPanel {
+  content: ReactNode
+  isOpen: boolean
+  id: string
+}
+
+const TabButton = ({ isOpen, label, id, onTabClick }: ITabButtonProps) => {
+  const buttonClass = classNames(styles.button, isOpen && styles.isActive)
+  return (
+    <button
+      role="tab"
+      className={buttonClass}
+      onClick={() => onTabClick(label)}
+      aria-selected={isOpen}
+      aria-controls={`tab-panel-${id}`}
+      id={`tab-button-${id}`}
+    >
+      {label}
+    </button>
+  )
+}
+
+const TabPanel = ({ content, isOpen, id }: ITabPanel) => {
```

Comment:
Change content to `content` to `children`, then handle declaratively (see next comment)